server/index.js

```diff
@@ -1,18 +1,38 @@
 const express = require('express');
 const cors = require('cors')
+const petModel = require('./db/index');
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 8000;
 
 app.use(cors())
 app.use(express.json())
 
+mongoose.connect('mongodb://localhost:27017/pet-cards', () => console.log('Connected to DB'))
+
 const petList = [];
 
 app.post('/api/pet', (req, res) => {
     const petInput = req.body;
-    petList.push(petInput)
-    console.log(petList)
+    petList.push(petInput);
```

Comment:
we we need petList if we using mongo?

Reply:
Forgot to delete it after I add MongoDB, deleted.